aptos-move/framework/aptos-framework/sources/reconfiguration.move

```diff
@@ -113,6 +114,7 @@ module aptos_framework::reconfiguration {
 
         // Call stake to compute the new validator set and distribute rewards.
         stake::on_new_epoch();
```

Comment:
Unrelated to this but are we supposed to call `stake::on_new_epoch()` here? @movekevin

Reply:
Yep this is correct. We need to recompute the validator set when there's a reconfig